File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h, r, c, s;
    do
    {
        h = get_int("Height number: ");
    }
    while (h < 1 || h > 8);

    for (r = 0; r < h; r++)
    {
        for (s = 0; s < h - r - 1; s++)
        {
            printf(" ");
        }
        for (c = 0; c <= r; c++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: Week 6/mario.py
#from cs50 import get_int

#for i in range(n):
    #print("#")

def main():
    height = get_height()
    for i in range(height):
        print("#")

def get_height():
    #instead of "do while" loop:
    while True:
        try:
            n = int(input("Height: "))
            if n > 0:
                return n
        except ValueError:
            print("Not a number")

main()